### PR TITLE
[DOC] Add NeuroStars question link

### LINKED ISSUE #647: Add NeuroStars link to issue template selection page
### Summary
I think we could benefit from having specific bug report and feature request issue templates, along with links for usage questions to NeuroStars. 

### Additional Detail
GitHub lets you link to external sites from your issue template selection page. We recently adopted this in `nilearn` and `NiMARE` (see `nilearn`'s page below):

image

We can have the NeuroStars link point directly to `tedana`-labeled topics, and when you create a new topic from that page, it automatically adds a `tedana` label.

### Next Steps
1. Add goal-specific issue templates (with autolabeling!)
2. Add link to NeuroStars `tedana` label page.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #647.

Changes proposed in this pull request:

- Add link to NeuroStars to issue template selection page.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+contact_links:
+    - name: Usage question
+      url: https://neurostars.org/tag/tedana
+      about: Please ask question about using tedana here.
```
Comment (jbteves) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
      about: Please ask questions about using tedana here.
```

**Review by jbteves — CHANGES_REQUESTED** [review]
Singular vs. plural typo; once that's corrected I'll approve.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (tsalo) [reply .github/ISSUE_TEMPLATE/config.yml:4]: Thx!

**Review by jbteves — APPROVED** [review]
LGTM, thanks for putting this in @tsalo ! That should make it much easier for new users.

**Review by eurunuela — APPROVED** [review]
LGTM!

**Pull request merged**